decosjoin/api/decosjoin/decosjoin_connection.py

```diff
@@ -180,11 +178,10 @@ def _transform(self, zaken, user_identifier):  # noqa: C901
                     {"name": "caseType", "from": "text45", "parser": to_string},
                     {"name": "identifier", "from": 'mark', "parser": to_string},
                     {"name": "dateRequest", "from": "document_date", "parser": to_date},
-                    {"name": "processed", "from": "processed", "parser": to_bool},
                     {"name": "dateDecision", "from": "date5", "parser": to_datetime},  # Datum afhandeling
                     {"name": "location", "from": "text6", "parser": to_string},
-                    {"name": "status", "from": "title", "parser": to_string},
-                    {"name": "decision", "from": "dfunction", "parser": to_string},
+                    {"name": "status", "from": "title", "parser": to_vakantie_verhuur_vergunning_status},
```

Comment:
beetje overkill, maar werkt wel.

Reply:
Wat is overkill? De naam, de functionaliteit? Het aantal `_`? :-)